### PR TITLE
[*] CORE : Add error log if can't write class_index

### PR DESCRIPTION
**Pull request by jmaumene**
Add message in error_log if can't write cache/class_index.php

### CODE REVIEW

File: classes/PrestaShopAutoload.php [hunk 1]
```diff
@@ -154,7 +154,10 @@ public function generateIndex()
 		if ($filename_tmp !== false && file_put_contents($filename_tmp, $content) !== false)
 		{
 			if (!@rename($filename_tmp, $filename))
+			{
```
Comment (tchauviere) [inline classes/PrestaShopAutoload.php]: Hi @jmaumene,

Can you please make it PSR2 compliant by pulling this brackets next to the end of prior "if" statement ?

Thanks :)

Comment (jmaumene) [inline classes/PrestaShopAutoload.php]: Hi tchauviere,

this is solved
I have not changed the tabs in the space , to not break the indentation file.

**Comment by vTerenti** [comment]
(note: this is an automated message, but answering it will reach a real human )

Hi! Thank you for your pull request: your contribution to PrestaShop means a lot to us!

We are aware that this PR is still open after all this time, and we want to make sure that your hard work is not forgotten.

We are diving back into pull requests, and we want to work with you in order to get your PR to completion: we need your feedback!<br/>
Could you read [this article](http://build.prestashop.com/news/review-your-pull-requests/) from [the Build devblog](http://build.prestashop.com/), and take the time to review your pull requests, so that we can work with up-to-date information? You don't have to do it all right now, and every little step helps.

Again, thank you very much for this contribution, and for your renewed feedback on it!

**Comment by vTerenti** [comment]
Hello!

This PR has been automatically closed because the branch 1.6 does not exist anymore.

Best regards

**Pull request closed without merging**